src/components/Modal.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+const Modal = ({title, children, onExitClick}) => {
+  return (
+    <div>
+      <div class="modal is-active">
```

Comment:
I think that for JSX, we need to use className rather than class. At least, I'm getting warnings in the console about this.

Reply:
Yeah I get those also every once in a while so definitely something we may need to unify on.